_posts/databases/postgresql/timescaledb/2000-01-01-intro.md

```diff
@@ -0,0 +1,20 @@
+---
+title: TimescaleDB
+nav: Introduction
+modified_at: 2022-01-21 00:00:00
+tags: timescale databases postgresql extensions
+index: 1
+---
+
+TimescaleDB is a PostgreSQL extension for the support of the time-series data.
+This extension add several features and functions, you can find more information
+on their documentation: [https://docs.timescale.com/api/latest](https://docs.timescale.com/api/latest).
+
+There are multiple licences of TimescaleDB, __Community edition__ and
+__Open-source edition__. The latter is the only one available on Scalingo, hence
+TimescaleDB's functions labeled "community" on their documentation
+are not available on the platform.
```

Comment:
Maybe this introduction page should talk about the two other pages created, and link to them?